### PR TITLE
[ICDS Dashbaord] Spelling corrections in infantomter and stadiometer

### PR DESCRIPTION
**Pull request by challabeehyv**
https://dimagi-dev.atlassian.net/browse/QA-898

##### SUMMARY
Spelling corrections in infantomter and stadiometer

### CODE REVIEW

File: custom/icds_reports/static/js/directives/kpi/kpi.directive.js [hunk 1]
```diff
@@ -24,6 +24,10 @@ function KpiController($location, haveAccessToFeatures) {
     };
 
     this.isNumber = window.angular.isNumber;
+
+    this.toShowInKpi = function (cellLabel) {
```
Comment (czue) [inline custom/icds_reports/static/js/directives/kpi/kpi.directive.js]: Could you add a comment that this is to be removed after QA?

**Review by czue — APPROVED** [review]
Looks good, just requested you add one comment

**Review by Rohit25negi — APPROVED** [review]
Looks good. Good to go after Cory's comment is addressed.

**Comment by Rohit25negi** [comment]
@challabeehyv  can you resolve the merge conflict and address Cory's comment

File: custom/icds_reports/static/js/directives/kpi/kpi.directive.js (hunk 1 above)
Comment (challabeehyv) [reply custom/icds_reports/static/js/directives/kpi/kpi.directive.js]: i beleive once verified, there will be one more ticket created to remove this from feature flag.Anyways i will add it

**Comment by challabeehyv** [comment]
closing this as a duplicate to https://github.com/dimagi/commcare-hq/pull/26132

**Pull request closed without merging**